themes/bootstrap3/js/common.js

```diff
@@ -378,7 +378,7 @@ var VuFind = (function VuFind() {
    * @param {string|JQuery} container
    */
   var initResultScripts = function initResultScripts(container) {
-    let jqContainer = typeof container === 'string' ? $(container) : container;
+    let jqContainer = typeof container === 'string' ? document.querySelector(container) : container;
```

Comment:
Is this an unintentional change? Changing the value of jqContainer seems unrelated to the rest of this pull request... and if we do this, we would both need to change the variable name (since it is no longer a jQuery container -- so the jq no longer means anything) and also verify whether all the init methods receiving jqContainer work properly with non-jQuery sets.

Reply:
This is from one of my other changes, sorry for overseeing it!